OpenRA.Mods.Common/Traits/Player/ClassicProductionQueue.cs

```diff
@@ -120,15 +120,15 @@ protected override bool BuildUnit(ActorInfo unit)
 
 		public override int GetBuildTime(string unitString)
 		{
-			return GetBuildTime(self.World.Map.Rules.Actors[unitString]);
+			return GetBuildTime(self.World.Map.Rules.Actors[unitString], self.World.Map.Rules.Actors[unitString].TraitInfo<BuildableInfo>());
 		}
 
-		public override int GetBuildTime(ActorInfo unit, BuildableInfo bi = null)
+		public override int GetBuildTime(ActorInfo unit, BuildableInfo bi)
```

Comment:
You'll have to fix https://github.com/OpenRA/OpenRA/pull/11311/files#diff-910e0462c780643261207e2829e4041dR135 then as well (we don't need to check for `bi` being `null` anymore).